wandb/filesync/step_prepare.py

```diff
@@ -109,11 +89,17 @@ def _gather_batch(
         self, first_request: RequestPrepare
     ) -> Tuple[bool, Sequence[RequestPrepare]]:
         batch_start_time = time.time()
```

Comment:
Could you replace this (and in the relevant other places) with either monotonic or perf_counter's?

Reply:
Monotonicized!